employdat-app/routes/web.php

```diff
@@ -22,17 +22,17 @@
 Route::get('/order/email', [dats_controller::class, 'orderEmail'])->name('order_email');
 Route::get('/order/post', [dats_controller::class, 'orderPost'])->name('order_post');
 Route::get('/search', [dats_controller::class, 'search'])->name('search_name');
-Route::get('/insert',function(){return view('insert');})->name('insert');
+Route::get('/insert',function(){return view('insert');})->name('insert')->middleware('auth');
 Route::post('/', [dats_controller::class, 'store'])->name('add_row');
-Route::delete('/{id}',[dats_controller::class,'delete'])->name('remove_row');
-Route::get('/confirm{id}',[dats_controller::class,'confirm'])->name('delete_confirm');
-Route::get('/edit{id}',[dats_controller::class,'edit'])->name('edit');
+Route::delete('/{id}',[dats_controller::class,'delete'])->name('remove_row')->middleware('auth');
+Route::get('/confirm{id}',[dats_controller::class,'confirm'])->name('delete_confirm')->middleware('auth');
+Route::get('/edit{id}',[dats_controller::class,'edit'])->name('edit')->middleware('auth');
 Route::patch('/{id}',[dats_controller::class,'update'])->name('update');
 
 
 Route::get('/cv{id}',[dats_cv_controller::class,'index'])->name('cvs');
```

Comment:
Please use a Resource Controller for this routes. https://laravel.com/docs/10.x/controllers#resource-controllers